components/MenuBar/MenuBar.jsx

```diff
@@ -0,0 +1,76 @@
+import { PropTypes, Component } from 'react'
+
+require('./MenuBar.scss')
+
+class MenuBar extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {mobile: false}
+  }
+
+  componentWillMount() {
+    this.handleResize(this)
+    const handleResize = this.handleResize
+    const thisRef = this
+    this.state.handler = function() { handleResize(thisRef) }
+    window.addEventListener('resize', this.state.handler)
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.state.handler)
+  }
+
+  handleResize(thisRef) {
+    let breakPoint = thisRef.props.mobileBreakPoint
```

Comment:
We should also be able to remove `thisRef` as an argument to `handleResize` and just use `this.props.propertyName`